### ISSUE TITLE
PHP JSON EXTENSION IS MISSING

### ISSUE THREAD

**Issue by iamsajidjaved** [stackery/php-lambda-layer#57]
![image](https://user-images.githubusercontent.com/32770676/66772577-767fa680-eed6-11e9-8314-a88304c8a3e4.png)

Seeing this error. I followed the documentation and created php.ini  at the root of my function package with:
extension=json.so

But still facing same issue. Can anyone tell me why I'm seeing this?

**Comment by txase** [comment]
It looks to me like this is looking for a json_decode() function provided by the Illuminate Foundation module, not the base PHP json module. Do you need to add the Illuminate Foundation module extension to your php.ini file?

**Comment by iamsajidjaved** [comment]
@txase I don't think so. It's because of missing extension. phpinfo() also doesn't show the json extension.

**Comment by iamsajidjaved** [comment]
@txase where is the old ARN php7.3? that was working fine
